### PR TITLE
completed MVP on JavaScript-II

### PR DESCRIPTION
**Pull request by marianahuh**
Submitting MVP JavaScript-II Project.

### CODE REVIEW

**Comment by marianahuh** [comment]
I changed the forEach parameter.

File: assignments/callbacks.js [hunk 1]
```diff
@@ -24,25 +31,52 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
...
+
+multiplyNums(3, 3, multiply);
+
 function contains(item, list, cb) {
   // contains checks if an item is present inside of the given array/list.
   // Pass true to the callback if it is, otherwise pass false.
+  return item.some(itemVal => list === itemVal);
 }
 
+console.log(contains(items, "yo-yo"));
```
Comment (ielvisd) [inline assignments/callbacks.js:78]: Nice contains function and great use of console.logs throughout to check your work.

File: assignments/array-methods.js [hunk 2]
```diff
@@ -2,82 +2,112 @@
...
-{"id":44,"first_name":"Jodie","last_name":"Stawell","email":"jstawell17@yale.edu","shirt_size":"S","company_name":"Jaxspan","donation":262},
-{"id":45,"first_name":"Falito","last_name":"Karsh","email":"fkarsh18@pcworld.com","shirt_size":"S","company_name":"Mycat","donation":239},
-{"id":46,"first_name":"Reginauld","last_name":"Purselowe","email":"rpurselowe19@thetimes.co.uk","shirt_size":"L","company_name":"Jabbersphere","donation":11},
-{"id":47,"first_name":"Vida","last_name":"Tydd","email":"vtydd1a@dropbox.com","shirt_size":"S","company_name":"Quaxo","donation":55},
-{"id":48,"first_name":"Anderea","last_name":"MacGiolla Pheadair","email":"amacgiollapheadair1b@xing.com","shirt_size":"2XL","company_name":"Kwimbee","donation":214},
-{"id":49,"first_name":"Bel","last_name":"Alway","email":"balway1c@ow.ly","shirt_size":"S","company_name":"Voolia","donation":107},
-{"id":50,"first_name":"Shell","last_name":"Baine","email":"sbaine1d@intel.com","shirt_size":"M","company_name":"Gabtype","donation":171}];
+const runners = [{ "id": 1, "first_name": "Charmain", "last_name": "Seiler", "email": "cseiler0@wired.com", "shirt_size": "2XL", "company_name": "Divanoodle", "donation": 75 },
```
Comment (ielvisd) [inline assignments/array-methods.js:5]: Great array methods!

File: assignments/closure.js [hunk 3]
```diff
@@ -1,11 +1,33 @@
...
+
 const counter = () => {
   // Return a function that when invoked increments and returns a counter variable.
+  let count = 0;
+  return function () {
+    return ++count;
+  }
 };
+
+const newCounter = counter();
```
Comment (ielvisd) [inline assignments/closure.js:25]: Nice counter function and console.logs to check.

**Review by ielvisd — APPROVED** [review]
Great work Mariana, keep it up.

**Pull request merged**